### PR TITLE
Add `Plugin load failed` message

### PR DESCRIPTION
**Pull request by mrxiaozhuox**
if plugin data is empty, display the message.

### CODE REVIEW

File: lapce-ui/src/plugin.rs [hunk 1]
```diff
@@ -158,6 +158,24 @@ impl Widget<LapceTabData> for Plugin {
...
+                let x = self.line_height;
+                let layout = ctx
+                    .text()
+                    .new_text_layout("Failed to load plugin information")
+                    .font(
+                        data.config.ui.font_family(),
+                        data.config.ui.font_size() as f64,
+                    )
+                    .default_attribute(TextAttribute::Weight(FontWeight::SEMI_BOLD))
+                    .text_color(Color::from_hex_str("#d29200").unwrap())
```
Comment (MinusGix) [inline lapce-ui/src/plugin.rs:172]: This should use a color from the theme. Likely `"lapce.warn"` or `"lapce.error"` would work.

**Comment by dzhou121** [comment]
Thanks! Looks good to me. Can you please run rustfmt and I'll merge.

**Comment by mrxiaozhuox** [comment]
> Thanks! Looks good to me. Can you please run rustfmt and I'll merge.

OK

**Comment by dzhou121** [comment]
Thanks!

**Pull request merged**